opendevin/runtime/docker/ssh_box.py

```diff
@@ -82,8 +83,19 @@ def __init__(
         self._ssh_port = find_available_tcp_port()
 
         # always restart the container, cuz the initial be regarded as a new session
-        self.restart_docker_container()
-
+        n_tries = 5
+        while n_tries > 0:
+            try:
+                self.restart_docker_container()
+                break
+            except Exception as e:
```

Comment:
Just curious what reason will result restart fail?

Reply:
For some reason, if you attempt to start multiple docker container in parallel, there's some slight chances that it will fail (not sure why 😅) - but retry generally fix it. 

I occasionally also see it when I try to 'make run' but it is not reproducible...